var/spack/repos/builtin/packages/intel-parallel-studio/package.py

```diff
@@ -86,7 +86,7 @@ def blas_libs(self):
         # TODO: TBB threading: ['libmkl_tbb_thread', 'libtbb', 'libstdc++']
         mkl_libs = find_libraries(
             mkl_integer + ['libmkl_core'] + mkl_threading,
-            root=join_path(self.prefix.lib, 'intel64'),
+            root=join_path(self.prefix, 'mkl', 'lib', 'intel64'),
```

Comment:
you surely can't. I am just saying to make things fast one could submit this change as a stand-alone PR, but still keep it here. Once a stand alone PR is merged, one can re-base this one on `develop`. 

But i don't really insist on this, just explain my rationale.